src/CLI/actioner/sonic-cli-if.py

```diff
@@ -98,6 +102,86 @@ def run(func, args):
     keypath, body = generate_body(func, args)
 
     try:
+        if func.__name__ == 'get_openconfig_if_ip_interfaces_interface_subinterfaces_subinterface_ipv4_neighbors':
```

Comment:
Should probably note somewhere, like in code comments, that this section is just a placeholder for now until the backend is implemented.